src/layer/tile/GridLayer.js

```diff
@@ -392,8 +392,8 @@ export var GridLayer = Layer.extend({
 
 			this._setZoomTransform(level, map.getCenter(), map.getZoom());
 
-			// force the browser to consider the newly added element for transition
-			Util.falseFn(level.el.offsetWidth);
+			// force reading offsetWidth so the browser considers the newly added element for transition
+			Util.falseFn(level.el.offsetWidth);	// eslint-disable-line
```

Comment:
Does the linting pass without this? Because if it does I'd rather not disable the linter here.
```suggestion
			Util.falseFn(level.el.offsetWidth);
```

Reply:
It passes without the comment.